digiwf-gateway/src/main/java/de/muenchen/oss/digiwf/gateway/configuration/SpaServerCsrfTokenRequestHandler.java

```diff
@@ -0,0 +1,41 @@
+package de.muenchen.oss.digiwf.gateway.configuration;
+
+import org.springframework.security.web.server.csrf.CsrfToken;
+import org.springframework.security.web.server.csrf.ServerCsrfTokenRequestAttributeHandler;
+import org.springframework.security.web.server.csrf.ServerCsrfTokenRequestHandler;
+import org.springframework.security.web.server.csrf.XorServerCsrfTokenRequestAttributeHandler;
+import org.springframework.web.server.ServerWebExchange;
+import reactor.core.publisher.Mono;
+
+public class SpaServerCsrfTokenRequestHandler extends ServerCsrfTokenRequestAttributeHandler {
```

Comment:
Könntest du hier nochmal in nem Kommentar auf der Refarch verweisen, dass man das weiß woher es ursprünglich kommt? 
Sonst denkt man sich vlt in nem Jahr. "Das geht auch einfacher" und baut das unnötig um.

Reply:
Hatte ich auch schon überlegt, dann ist halt der Kommentar aus der SecurityConfiguration zweimal vorhanden oder würde du ihn dann eher nur hier hin packen?

Reply:
nja da steht ja nur der Verweis auf spring security aber nicht auf die LHM RefArch "Festlegung".

Reply:
Hat ja auch nichts mit der Refarch zu tun, dass kommt nur rein aus der Spring Security Doku.
Aktuell ist es in der RefArch auch noch so wie davor. Diese Änderung wird dann danach in die RefArch übernommen